vectorhub/encoders/image/tfhub/mobilenet.py

```diff
@@ -55,14 +91,53 @@ def init(self, model_url: str):
         ])
         self.model.build([None, self.image_dimensions, self.image_dimensions, 3])
     
+    def _read(self, image: str):
```

Comment:
This change feels unnecessary. This means there is now 2 read functions to update, one from base one in mobile-net.

Reply:
Removed

Reply:
My initial intention was to introduce a bulk_read function to use with the bulk_encode function.